pkg/apis/openstack/types_infrastructure.go

```diff
@@ -40,6 +40,8 @@ type Networks struct {
 	Worker string
 	// Workers is a CIDRs of a worker subnet (private) to create (used for the VMs).
 	Workers string
```

Comment:
See comments in `pkg/apis/openstack/v1alpha1/types_infrastructure.go`

Reply:
see reply to said comment